test/integration/TrackerRegistry.test.js

```diff
@@ -0,0 +1,107 @@
+import { getTrackerRegistry } from '../../src/utils/TrackerRegistry'
+
+const contractAddress = '0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF'
+const jsonRpcProvider = 'http://localhost:8545'
+
+describe('TrackerRegistry', () => {
+    test('get array of trackers', async () => {
+        const trackerRegistry = await getTrackerRegistry({
+            contractAddress, jsonRpcProvider
+        })
+
+        expect(trackerRegistry.getAllTrackers()).toStrictEqual([
+            'ws://10.200.10.1:30301',
+            'ws://10.200.10.1:30302',
+            'ws://10.200.10.1:30303'
+        ])
+    })
+
+    test('throw exception if address is wrong (ENS)', async (done) => {
```

Comment:
FYI `async (done)` is going away when `jest-circus` lands:
https://github.com/facebook/jest/pull/9129

Scheduled for Jest 27:
https://github.com/facebook/jest/issues/6295

I opened a related issue here:
https://github.com/facebook/jest/issues/10529

Reply:
To get rid of the `done` the  equivalent would probably be this:
```js
await expect(await getTrackerRegistry({
  contractAddress: 'address', jsonRpcProvider
})).rejects.toThrow('Error: network does not support ENS')
```